src/lib/es2023.array.d.ts

```diff
@@ -20,7 +26,51 @@ interface Array<T> {
      * @param thisArg If provided, it will be used as the this value for each invocation of
      * predicate. If it is not provided, undefined is used instead.
      */
-    findLastIndex(predicate: (value: T, index: number, array: T[]) => unknown, thisArg?: any): number;
+    findLastIndex(
+        predicate: (value: T, index: number, array: T[]) => unknown,
+        thisArg?: any
+    ): number;
+
+    /**
+     * Copies the array and returns the copied array with all of its elements reversed.
+     */
+    toReversed(): T[];
+
+    /**
+     * Copies and sorts the array.

```

Comment:
```suggestion
     * Sorts and returns a copy of an array.
```

Reply:
@sandersn's suggested `"Sorts and ..."` language seems to imply an in-place mutation, followed by the return of a copy of that mutated array. But the `.to_` methods are written to not mutate the original array at all, and only mutates newly created copy at most (if at all).

For maximal consistency, we can instead use the exact verbiage suggested above for `toReversed()`, but substituting `s/reversed/sorted/`.

```suggestion
     * Returns a copy of an array with its elements sorted.
```